src/components/atoms/InputField/InputField.tsx

```diff
@@ -61,9 +61,20 @@ const InputField: React.ForwardRefRenderFunction<
 
   const inputClassNames = classNames("input-field__input", inputClassName);
 
+  const keyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === "Escape" && e.target) {
```

Comment:
Can you please also add this `"Escape"` to `KeyboardShortcutKeys` in `src/settings`. 

Probably under a 'root key/namespace' of `inputField` or similar, then something like `blur` or `unfocus` or `deselect` or some other word that describes the intent of this functionality in a succinct yet descriptive way, then use that here rather than the string literal.

Reply:
Defined `.escape` in `KeyboardShortcutKeys` in f79f6bc94dd969fc81493962b77da308cda7c557 . Not sure yet what you meant by the rest of the comment.

Reply:
> Not sure yet what you meant by the rest of the comment.

@yarikoptic Instead of:
```typescript
export const KeyboardShortcutKeys = {
  chat: "c",
  escape: "escape",
```

Something like (or one of the other words mentioned in place of `inputField` and `deselect` if you think a different one makes more sense):

```typescript
export const KeyboardShortcutKeys = {
  chat: "c",
  inputField: {
    deselect: "escape",
  }
```

Reply:
done. force-pushed with rebase on top of current staging (conflicts start to appear, pain, suffering...)